spec/lib/swagger/rake_swagger_spec.rb

```diff
@@ -135,12 +138,15 @@ def expect_response_params_def(http_method, path, response_code, param_name, fie
         expect_param_def("get", "/users/by_department", "department", "enum",
                          %w[finance operations sales marketing HR])
 
+        expect_param_def("get", "/users/by_department_number", "department", "type", "integer")
+        expect_param_def("get", "/users/by_department_number", "department", "enum", [1, 2, 3])
+
         expect_param_def("get", "/users/in_departments", "departments", "in", "query")
         expect_array_param_def("get", "/users/in_departments", "departments",
                          %w[finance operations sales marketing HR])
 
         expect_tags_def("get", "/twitter_example/{id}/followers", %w[twitter_example following index search])
-
+        expect_response_params_def("get", "/pets/{id}/as_properties", 200, "pet_name", "example", "mypet")
```

Comment:
this added line is not related to this PR. i've just copied it from the other changed example above because it looks as if the two examples were supposed to mirror each other.